cg/store/filters/status_case_filters.py

```diff
@@ -32,7 +32,7 @@ def get_inactive_analysis_cases(cases: Query, **kwargs) -> Query:
 
 def get_active_cases(cases: Query, **kwargs) -> Query:
     """Return cases which are running."""
-    return cases.filter(Family.action == "running")
+    return cases.filter(Family.action == CaseActions.RUNNING)
```

Comment:
I think the actions are analyse, running hold. 
We could be more explicit here as in:

`get_running_cases`